tests/test_cli.py

```diff
@@ -0,0 +1,127 @@
+"""Test for checking Cli"""
+import unittest
+import random
+import string
+import json
+import re
+import uuid
+from hydrus.app_factory import app_factory
+from hydrus.utils import set_session, set_doc, set_api_name
+from hydrus.data import doc_parse, crud
+from hydra_python_core import doc_maker
+from hydrus.samples import doc_writer_sample
+from sqlalchemy import create_engine
+from sqlalchemy.orm import sessionmaker, scoped_session
+from hydrus.data.db_models import Base
+import click
+from click.testing import CliRunner
+from cli import startserver
+
+def test_startserver():
```

Comment:
This function is never called/used. What is it here for?

Reply:
He meant to make it a part of the test class but it's outside the scope. Would be wonderful if you could share a good resource to learn testing as we are new to it.

Reply:
@chrizandr  yeah i forget to put in test class now i changed ,will make new PR

Reply:
@knakul853 don't make a new PR, you can always undo a commit and force push the changes.

Reply:
@knakul853 kindly take a look at the docs
unittest -> https://docs.python.org/3/library/unittest.html
unittest.mock -> https://docs.python.org/3/library/unittest.mock.html

Reply:
@ShravanDoda  thanks